report/2.Analysis/RTSIntroduction.tex

```diff
@@ -21,7 +21,8 @@ \subsection{Schedulability}
 Likewise, it is relevant to consider the order in which the different tasks are executed, and estimate which tasks have the highest level of relevance.
 
 \subsection{Takeaway}
-Looking into how to optimize the different processes is important.
+Looking into how to optimize the different processes is important and will be elaborated upon in Section~\ref{Theory:RTS}..
 Without considering the order in which to do tasks, and the expected time frame of a task, it is impossible to interact with a moving targets and a changing world. 
 Therefore, the principles of real-time systems, are quite relevant to the problems of this project.
-
+Another major obstacle of the project is the process of tracking objects.
+%TODO Figure out how to go from real time to object tracking
```

Comment:
This transition is kinda cheesy, but I think it'll do

Reply:
tror ikke vi behøver en direkte overgang når det er "takeway". det er slutningen på noget. så en given overgang skal være i starten af næste, eller også er det bare ret "implicit okay" overgang